### PR TITLE
Revised flag help hints

### PR DESCRIPTION
**Pull request by olliephillips**
I felt the flag help hints could be improved. I've also edited the English of the README. 

For the repo title, I would use: 

#### Reverse proxy which automatically obtains TLS certificates from Let's Encrypt

Hope it helps.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -43,51 +44,51 @@ const (
...
 	initOnly                      = flag.Bool("init-only", false, "Exit after initialize, generate self keys. Need for auto-test environment.")
 	keepAliveModeS                = flag.String("keepalive", KEEPALIVE_TRANSPARENT_STRING, KEEPALIVE_TRANSPARENT_STRING+" - keepalive from user to server if both support else doesn't keepalive. "+KEEPALIVE_NO_BACKEND_STRING+" - force doesn't use keepalive connection to backend, but can handle keepalive from user.")
-	keepAliveCustomerTimeout      = flag.Duration("keepalive-customer-timeout", time.Minute*15, "When keepalive in mode '"+KEEPALIVE_NO_BACKEND_STRING+"' - how long time keep customer's connection. In '"+KEEPALIVE_TRANSPARENT_STRING+"' mode timeout don't used and both connection close when one of backend or customer close self connection.")
+	keepAliveCustomerTimeout      = flag.Duration("keepalive-customer-timeout", time.Minute*15, "When keepalive in mode '"+KEEPALIVE_NO_BACKEND_STRING+"' - how long should the connection be maintained. In '"+KEEPALIVE_TRANSPARENT_STRING+"' mode, `timeout` isn't used and both connections close when either the backend or customer close self connection.")
```
Comment (rekby) [inline main.go]: Why `timeout` need backquotes?

File: main.go [hunk 2]
```diff
@@ -43,51 +44,51 @@ const (
...
-	logrotateTime                 = flag.String("logrotate-time", "", "minutely|hourly|daily|weekly|monthly|yearly|\"\", empty or none mean no logrotate by time. Weekly - rotate log at midnight from sunday to monday")
-	minTLSVersion                 = flag.String("min-tls", "", "Minimul supported tls version: ssl3,tls10,tls11,tls12. Default is golang's default.")
-	noLogStderr                   = flag.Bool("no-log-stderr", false, "supress log to stderr")
-	nonCertDomains                = flag.String("non-cert-domains", "", "No obtain certificate for mathed domains. Regexpes separated by comma.")
-	pidFilePath                   = flag.String("pid-file", "lets-proxy.pid", "Write pid of process. When used --daemon - lock the file for prevent double-start daemon.")
-	preventIDNDecode              = flag.Bool("prevent-idn-decode", false, "Default domain show in log as 'domain.com' or 'xn--d1acufc.xn--p1ai' ('домен.рф'). When option used it will shown as 'domain.com' or 'xn--d1acufc.xn--p1ai', without decode idn domains.")
+	logrotateMaxCount             = flag.Int("logrotate-count", 30, "How many old backups to keep. 0 for keep indefinitely.")
```
Comment (rekby) [inline main.go]: I think abount word infinite or unlimited.
I saw indefinitely dictionaris and examples in godoc: indefinitely more like to undefined, then unlimited.

File: main.go (hunk 1 above)
Comment (olliephillips) [inline main.go]: Happy to remove the quotes. Maybe unnecessary as you say.

File: main.go (hunk 2 above)
Comment (olliephillips) [inline main.go]: I would stay with indefinitely vs infinitely on this one, but when I read it again maybe 'forever' is much better than both of these options?

Comment (rekby) [inline main.go]: No, forever is bad, becouse log rotation have other option: logrotate-age and old logs may be unlimited by number, but delete old logs by age.

if indefinitely and infinitely is one I think infinite is simpler word.

Comment (olliephillips) [inline main.go]: Happy to revert PR to infinitely if this best describes that option. What about "0 for keep all"?

Comment (rekby) [inline main.go]: I think keep all but - some reason as forever.
May by infinitely, unlimited count, disable count limit, no count limit? With accent for unlimited by count only But not for disable remove old logs complete.

Comment (olliephillips) [inline main.go]: I'll revert it to "infinitely" - it is semantic, and it wasn't my intention to take up your time on this. I'll get the commits included in the PR in the next 10 minutes

**Comment by rekby** [comment]
Thanks for your help :)

**Pull request merged**